src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private final Colors color;
+    private final int number;
+
+    Ball(Colors color, int number) {
```

Comment:
Why constructor package-private? Is there are any reason for that?

Reply:
not fixed. Add access modifier